index.html

```diff
@@ -102,24 +102,24 @@ <h1 id = "page-title" class = "drop-shadow-2xl text text-center font-Rowdies tex
             <br>
             <button id="history-button" class=" dark:md:hover:bg-yellow-50 p-1.5 bg-orange-300 text-black font-thin">View Historical Results</button>
             <br>
-            <button id="clear-history">clear history</button> <!--used for testing-->
+            <button id="clear-history">clear history</button> 
         </div>
         </section>
     <br>
     
-        <!-- Current Results -->
-        <section>
```

Comment:
@Julian-A-Mort any reason why this section was removed? Looks like we would lose the youtube preview if we do